test/integration/RoutePatternTest.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const TestHelper = require('../TestHelper');
+const mockyeah = TestHelper.mockyeah;
+const request = TestHelper.request;
+
+describe('Route Patterns', () => {
+  it('should work with route parameter', (done) => {
+    mockyeah.get('/service/:key');
```

Comment:
I favor "dynamic segment" over "parameter" language, I hear parameter and I think URL parameters. Just a tidbit.

Reply:
@ryanricard I tend to agree, however, Express calls these "parameters" - see http://expressjs.com/en/guide/routing.html#route-parameters.

Reply:
We do not need to be bound to express's choice of language.